bin/vercel-ignore-step.sh

```diff
@@ -2,6 +2,9 @@
 
 echo "VERCEL_GIT_COMMIT_REF: $VERCEL_GIT_COMMIT_REF"
 
-# Don't build
-echo "🛑 - Build cancelled"
-exit 0;
+if [ "$VERCEL_GIT_COMMIT_REF" = "assets" ]; then
+  echo "Ignoring branch $VERCEL_GIT_COMMIT_REF"
+  exit 0
+fi
+
+yarn dlx turbo-ignore
```

Comment:
```suggestion
yarn dlx turbo-ignore

```

just adding a new line